recipes/openff-toolkit/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "openff-toolkit" %}
+{% set version = "0.9.0rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/openff-toolkit/archive/{{ version }}.tar.gz
+  sha256: 38eeab18f5e2c0eeeb06d96ef93397070319a7e55fc9ea61d806f45c43f82cdb
+
+build:
+  noarch: python
+  number: 0
```

Hm, I think you don't need a separate `build.sh`. You can do this:
```suggestion
  number: 0
  script: "{{ PYTHON }} -m pip install . -vv; exit 1"
```